### PR TITLE
Add visual taps to arecord

### PR DESCRIPTION
**Pull request by hasansidd**

### CODE REVIEW

File: arecord [hunk 1]
```diff
@@ -15,10 +15,16 @@ echo "📁 Copying the video from device..."
 adb -s $SELECTED_DEVICE pull /mnt/sdcard/output.mp4 ~/Desktop/$FILENAME.mp4 &> /dev/null
 adb -s $SELECTED_DEVICE shell rm /mnt/sdcard/output.mp4 &> /dev/null
 echo "✅ Done"
+echo " Turning off taps..."
```
Comment (IntergalacticPenguin) [inline arecord]: I would like to remove the added echo output, otherwise great job 👍

File: arecord [hunk 2]
```diff
@@ -15,10 +15,16 @@ echo "📁 Copying the video from device..."
...
+adb -s $SELECTED_DEVICE shell settings put system show_touches 0
+echo "✅ Done"
```
Comment (IntergalacticPenguin) [inline arecord]: This too.

File: arecord [hunk 3]
```diff
@@ -15,10 +15,16 @@ echo "📁 Copying the video from device..."
...
+
 exit
 }
 
 android_choose_device
 
 echo "📹 Recording screen on $SELECTED_DEVICE, stop it with ctrl^c"
+echo "👇 Showing taps"
```
Comment (IntergalacticPenguin) [inline arecord]: And this 🙏

**Comment by hasansidd** [comment]
Echos have been removed

**Pull request merged**